lib/logstash/inputs/elasticsearch.rb

```diff
@@ -262,22 +268,39 @@ def run(output_queue)
   end
 
   private
-
+  JOB_NAME = "run query"
   def do_run(output_queue)
     # if configured to run a single slice, don't bother spinning up threads
-    return do_run_slice(output_queue) if @slices.nil? || @slices <= 1
+    return retryable(JOB_NAME) do
```

Comment:
I think this, with the addition of `retryable` decorator, has switched to a not trivial block and  hit a [style rule](https://github.com/rubocop/ruby-style-guide#multi-line-if-modifiers).
To me is more obscure  than:
```ruby
if @slices.nil? || @slices <= 1
  return retryable(JOB_NAME) do
    do_run_slice(output_queue)
  end
end
```

Reply:
I agree `retries` is a better word, more familiar to users and aligns the experience with other plugins. Thanks for calling a change!

Reply:
When I was writing `tries`, I have an impression that `retries` is confusing because I always need to check if it includes the first run. Probably, I am the only one who feels confused.